### PR TITLE
Added a document to define metrics. UI will use it on metrics.

### PR DESCRIPTION
**Pull request by yuyiguo**
@vkuznet please take a look.

### CODE REVIEW

File: src/maps/Doc4Metrics.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+"IO":{
```
Comment (vkuznet) [inline src/maps/Doc4Metrics.py]: please change IO to storage, it would be better to follow the schema.

File: src/maps/Doc4Metrics.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "readAverageKB": "Read rate of storage layer in KB/operation.",
+    "readTotalMB": "Total storage layer read in MB.", 
+    "readNumOps": "Storage-based read operation",
+    "readCachePercentageOps": "% successful operations by ROOT layer vs storage.", 
+    "readMBSec": "Read rate MB/millisecond.",
+    "readMaxMSec": "The longest read time in millisecond.",
+    "readTotalSecs": "Total time for root-based read in seconds.",
+    "writeTotalSecs": "Total writting time in seconds."
+            },
+"CPU":{
```
Comment (vkuznet) [inline src/maps/Doc4Metrics.py]: use lower case to match the schema

File: src/maps/Doc4Metrics.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+"CPU":{   
+    "MinEventTime":"Shortest event wall time in seconds.",
+    "MaxEventTime":"Longest event wall time in seconds",
+    "AvgEventTime":"Average event wall time in seconds.",
+    "EventThroughput":"Num of events per second.",
+    "TotalJobTime":"Job wall time in seconds.",
+    "TotalJobCPU":"Job CPU in seconds.",
+    "TotalLoopCPU":"in seconds."
+},
+"memory":{
```
Comment (vkuznet) [inline src/maps/Doc4Metrics.py:23]: memory is part of storage, please follow the schema.

Comment (yuyiguo) [reply src/maps/Doc4Metrics.py:23]: memory is not part of storage. It is a separated part parallel to storage.  The schema I use is https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Services/WMArchive/DataMap.py#L39. If not,  please show me which schema.

**Comment by yuyiguo** [comment]
Done all the changes.

**Comment by vkuznet** [comment]
Yuyi,
you're right, memory is separate from storage, I was mistaken by bad
identination over here:
https://github.com/dmwm/WMArchive/blob/master/src/python/WMArchive/Schemas/FWJRProduction.py#L90
Valentin.

**Pull request merged**